### DISCUSSION TITLE
Anycubic Mono M7 Max

### DISCUSSION THREAD

**Discussion by Axsom** [sn4k3/UVtools#960] [General]
Hello, I just upgraded my Mono X to the Mono M7 Max. Since it's still a newer machine, I wasn't surprised that you haven't added it yet.
Would you happen to have any plans to add it?
Is there anything I can provide to assist?
I've seen some of the information others had given you for the M7 & M7 Pro, I have to be honest, I'm not sure where to retrieve those files. If you let me know what you need I will find it for you.

**Comment by sn4k3** [comment]
Hello,

Update your UVtools.
In last release new anycubic machines are supported.
